package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "test": "mocha test --require babel-core/register --require babel-polyfill --exit --timeout 5000 ./test/**/**/*.test.js",
-    "start": "babel-node index.js",
-    "build": "babel server -d dist",
+    "start": "npm run build && node dist/app.js",
+    "build": "babel . --ignore node_modules,test,coverage,dist ../valinor-ah-backend -d dist/",
```

Comment:
I guess this will give us errors now as we have not merged @sulenchy's PR that has the index.js file renamed. 

What we can do here is to also add a script to remove the dist folder and recreate it everytime we run build. What this helps us do is ensure that files that have been renamed or deleted from the application do not still exist in the dist folder.

Something like this should work `"clean": "rm -rf dist && mkdir dist",`

and then for your build script, you can then do this

`"build": "npm run clean && babel . --ignore node_modules,test,coverage,dist ../valinor-ah-backend -d dist/",`

Reply:
ok will do